Adding precision on PANTHER_NO_HEADLESS behavior (#639)
symfony/panther: README.md

The following environment variables can be set to change some Panther's behaviour:

> * `PANTHER_NO_HEADLESS`: if defined and true, then it disables Panther's default use of headless mode with a 1200x1100 window (and disable GPU for chrome) 

Reviewer: this is documenting internal details that might change (see eg #640) so I'm not sure about the benefit of stating this :thinking: 

```suggestion
* `PANTHER_NO_HEADLESS`: set to `1` to disable Panther's use of headless mode and thus see what happens in a browser window
```

#### Chrome-specific Environment Variables

* `PANTHER_NO_SANDBOX`: to disable [Chrome's sandboxing](https://chromium.googlesource.com/chromium/src/+/b4730a0c2773d8f6728946013eb812c6d3975bec/docs/design/sandbox.md) (unsafe, but allows to use Panther in containers)

> * `PANTHER_CHROME_ARGUMENTS`: to customize Chrome arguments. You need to set `PANTHER_NO_HEADLESS` to a true value to have full control of arguments.

Reviewer: (same below)
```suggestion
* `PANTHER_CHROME_ARGUMENTS`: to customize Chrome arguments. You need to set `PANTHER_NO_HEADLESS` to `1` to have full control over arguments.
```
